polarCoordinateTraversal_old.m

```diff
@@ -109,6 +111,13 @@
     current_voxel_ID_r, circle_center, ...
     circle_max_radius, delta_radius, delta_radius, t, verbose);
 
+    if tMaxR < t_begin || tMaxR > t_end
```

Comment:
Instead of tMaxR > t_end, should it be tMinR > t_end? Also, where do you calculate tMaxR?

Reply:
There is no ```tMinR```. ```tMaxR``` is the distance until the next radial hit. It copies the fast voxel traversal algorithm's ```tMaxX``` style. But I agree, maybe a chance of variable name may be appropriate in the future.

```tMaxR``` is always calculated in ```radial_hit```. In this case, ```radial_hit_old```.